evm/src/interfaces/IWormholeTransceiverState.sol

```diff
@@ -50,7 +50,7 @@ interface IWormholeTransceiverState {
     /// @notice Set whether the chain is EVM compatible.
     /// @dev This function is only callable by the `owner`.
     /// @param chainId The Wormhole chain ID to set.
```

Comment:
Can you update the docstring to include the new param pls?

Reply:
yea, fixed.